ls1intum/Artemis
# Add new exercise type icon and tooltip to exam checklist table

### Checklist

### Motivation and Context


We display the exercise type and tooltip in the exam checklist exercise table.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. Exam -> Checklist -> Check if the exercise type is displayed correctly. 

### Screenshots

![grafik](https://user-images.githubusercontent.com/72132281/125329459-cdcdd780-e345-11eb-8121-a68ccfb8fbd9.png)
